### PR TITLE
fix(dx): alias schema to lookup helpers

### PR DESCRIPTION
**Pull request by smothers**
### Testing
- [ ] added relevant test coverage
- [x] no tests needed

### Docs
- [x] added relevant docs 
    - preview them at https://homebase.io/docs/homebase-react/{BRANCH_NAME}/overview
- [x] updated relevant sections in the README.md
- [x] updated relevant docstrings in index.d.ts
- [ ] no docs needed

### Typescript
- [x] added or edited relevant Typescript type declarations
- [ ] no type declaration updates needed

## Merging
For maintainers.

To merge, select "Squash and Merge". Then:
  1. Make sure the top commit message follows [Angular Git Commit Guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines),
  2. Delete all other commit messages in the description, but keep any lines designating [co-authors](https://docs.github.com/en/free-pro-team@latest/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors) so contributors will retain credit for their contributions.

### CODE REVIEW

**Comment by smothers** [comment]
FYI the Vercel previews of the todo examples will not work since they're pulling homebase-react from npm and `lookupHelpers` isn't published yet. They should work once this PR is merged. I made sure to update the package.json of each example to grab the latest version of homebase-react.

**Review by rubinovitz — APPROVED** [review]
Whoa amazing how much clearer this name change makes things when reading this all over again. No more having to wonder what kinds of "schema" we support vs not. Thanks for getting this in! LGTM!!

**Pull request merged**
